diff --git a/Bugzilla/Install/Requirements.pm b/Bugzilla/Install/Requirements.pm
index e653e5f..20026fd 100644
--- a/Bugzilla/Install/Requirements.pm
+++ b/Bugzilla/Install/Requirements.pm
@@ -404,14 +404,6 @@ sub OPTIONAL_MODULES {
         feature => ['memcached'],
     },

-    # BMO - metrics
-    {
-        package => 'ElasticSearch',
-        module  => 'ElasticSearch',
-        version => '0',
-        feature => ['elasticsearch'],
-    },
-
     # multi factor auth - totp
     {
         package => 'Auth-GoogleAuth',
