diff --git a/Bugzilla/Install/Requirements.pm b/Bugzilla/Install/Requirements.pm
index 22eb4f0..b1cac45 100644
--- a/Bugzilla/Install/Requirements.pm
+++ b/Bugzilla/Install/Requirements.pm
@@ -389,14 +389,6 @@ sub OPTIONAL_MODULES {
         version => '0',
         feature => ['memcached'],
     },
-
-    # BMO - metrics
-    {
-        package => 'ElasticSearch',
-        module  => 'ElasticSearch',
-        version => '0',
-        feature => ['elasticsearch'],
-    },
     );

     my $extra_modules = _get_extension_requirements('OPTIONAL_MODULES');
