diff --git a/mod_perl.pl b/mod_perl.pl
index 5840351..151004f 100644
--- a/mod_perl.pl
+++ b/mod_perl.pl
@@ -64,7 +64,7 @@ use Apache2::SizeLimit;
 if (Bugzilla->params->{'urlbase'} eq 'https://bugzilla.mozilla.org/') {
     Apache2::SizeLimit->set_max_unshared_size(700_000);
 } else {
-    Apache2::SizeLimit->set_max_unshared_size(250_000);
+    Apache2::SizeLimit->set_max_unshared_size(700_000);
 }

 my $cgi_path = Bugzilla::Constants::bz_locations()->{'cgi_path'};
