diff --git a/checksetup.pl b/checksetup.pl
index 501138c..0091ab5 100755
--- a/checksetup.pl
+++ b/checksetup.pl
@@ -230,7 +230,7 @@ Bugzilla::Install::reset_password($switch{'reset-password'})
 # Create default Product
 ###########################################################################

-Bugzilla::Install::create_default_product();
+#Bugzilla::Install::create_default_product();

 Bugzilla::Hook::process('install_before_final_checks', { silent => $silent });
